### PR TITLE
更新了文档

### PR DESCRIPTION
**Pull request by LycrusHamster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Suppose the [AssetService](https://github.com/nervosnetwork/muta/blob/master/bui
 6. send certain asset to another address
 
 ```js
-async function example(){
+test.skip('Muta example', async t => {
```
Comment (homura) [inline README.md]: test.skip is only for not trigger the test running, `async function` is fine

File: README.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ npm install muta-sdk@next
 
 ## Example
 
-Suppose the [AssetService](https://github.com/nervosnetwork/muta/blob/master/built-in-services/asset/src/lib.rs) is stable, the bellow code shows how to create a UDT(user define token) and how to transfer UDTs
+Suppose the [AssetServiceTs](https://github.com/nervosnetwork/muta/blob/master/built-in-services/asset/src/lib.rs) is stable, the bellow code shows how to create a UDT(user define token) and how to transfer UDTs
```
Comment (homura) [inline README.md]: Explicit difference between ts and js is unnecessary, recommend `AssetService`

Comment (LycrusHamster) [reply README.md]: IDE的锅 重构的时候 不小心搞到了...

File: src/service/binding.ts [hunk 3]
```diff
@@ -194,11 +213,20 @@ export type BindingClassPrototype<Binding> = {
     : never;
 };
 
+/**
+ * an alias
+ */
 type ServiceBindingClass<T> = new (
```
Comment (homura) [inline src/service/binding.ts]: The `ServiceBindingClass ` is not just an alias. the `type BindingClassPrototype ` would create an object like this

```ts
{
  transfer: (payload, privateKey? ) => Transaction | Receipt
}
```

and the  `type ServiceBindingClass ` would create a class(not an instance)

**Comment by homura** [comment]
muta-sdk has completely rewritten

**Pull request closed without merging**